plio/views.py

```diff
@@ -558,6 +558,34 @@ def duplicate(self, request, pk):
         item.save()
         return Response(self.get_serializer(item).data)
 
+    @action(methods=["delete"], detail=False)
```

Comment:
Add permissions?
item bulk deletion should be a protected action :p

Reply:
I was a little confused ki does adding a custom action inherit the permissions of the viewset or not? if it does, we won't need to add. will research

Reply:
check this actually

Reply:
the action inherits the permissions of the viewsets. the option given here is in the case if we want to override it.

Reply:
can you also do these quick changes in other viewsets as well? shouldn't take long